flit_core/flit_core/config.py

```diff
@@ -10,7 +10,10 @@
 try:
     import tomllib
 except ImportError:
-    from .vendor import tomli as tomllib
+    try:
+        import tomli as tomllib
+    except ImportError:
+        from .vendor import tomli as tomllib
```

Comment:
Is it OK for you if we swap round the logic here - i.e. try the vendored version first, and only load a separate tomli if the vendored one is removed?

My reason for preferring this is that it means there's no difference in behaviour between when tomli is installed or not.

Reply:
Yes. I understand the reasoning, and that works fine for us.